src/pipx/commands/common.py

```diff
@@ -222,6 +261,20 @@ def get_venv_summary(
         {add_suffix(name, package_metadata.suffix) for name in package_metadata.apps}
         - set(exposed_binary_names)
     )
+    exposed_man_paths = []
+    for i in range(1, 10):
+        man_section = "man%d" % i
+        exposed_man_paths += get_exposed_man_paths_for_package(
+            venv.man_path / man_section,
+            constants.LOCAL_MAN_DIR / man_section,
+            man_pages,
+        )
+    exposed_man_pages = sorted(
+        str(Path(p.parent.name) / p.name) for p in exposed_man_paths
+    )
```

Comment:
It gets slightly comlicated because `get_exposed_man_paths_for_package` is called from multiple places, among which is `_get_venv_resource_paths`. `_get_venv_resource_paths` is currently called for each `man<i>` directory separately. The reason is because it checks if a symlink can be created in the directory (`can_symlink(local_resource_dir)`), which shouldn't be done for the main man directory (`~/.local/share/man`), but for each `man<i>` subdirectory. `_get_venv_resource_paths` then calls `get_exposed_man_paths_for_package` indirectly through `_get_package_man_paths`. If the man_section loop is located in `get_exposed_man_paths_for_package`, there will have to be another man_section loop in `_get_venv_resource_paths` for the `can_symlink` checks. It is of course possible, but there isn't a neat and simple way of getting rid of the loops.

Another option how to make it slightly nicer would be to add a constant such as `MAN_DIRS = ["man%d" % i for i in range(1, 10)]` in `constants.py` and then loop over the list in other places.